geemap/common.py

```diff
@@ -3481,6 +3481,16 @@ def screen_capture(filename, monitor=1):
 ########################################
 
 
+def open_url(url):
+    """Opens the URL in a new browser tab."""
+    if in_colab_shell():
+        display(Javascript('window.open("{url}");'.format(url=url)))
```

Comment:
Should be: `window.open("{url}", "_blank", "noopener")` to open in a new tab and for security reasons:
https://www.js-craft.io/blog/window-open-and-target_blank-have-a-security-vulnerability

Reply:
Done. Thanks!